compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -521,7 +521,7 @@ internal expect fun pointerInputEvent(
 internal expect fun makeAccessibilityController(
     skiaBasedOwner: SkiaBasedOwner,
     component: PlatformComponent
-): AccessibilityController?
```

Comment:
Seems ability to not have a11y controller makes sense to keep.

Reply:
Nullability was a part of #106, and I've just revert it